scripts/build.sh

```diff
@@ -26,12 +25,12 @@ echo "◆ Generate FFI bindings"
 echo
 
 # See: https://cjycode.com/flutter_rust_bridge/integrate/deps.html#system-dependencies
-CPATH="$(clang -v 2>&1 | grep "Selected GCC installation" | rev | cut -d' ' -f1 | rev)/include" \
-        flutter_rust_bridge_codegen \
-                            --inline-rust \
-                            --skip-add-mod-to-lib \
-                            --rust-input packages/p2panda/native/src/api.rs \
-                            --dart-output packages/p2panda/lib/src/bridge_generated.dart
```

Comment:
I've added this to make the build work on my machine but that was with an older version of the flutter_rust_bridge, let me try if it builds without it now as well 🫣

Reply:
This is the link to the documentation (which got updated and is currently broken in the script):

https://cjycode.com/flutter_rust_bridge/v1/integrate/deps.html#system-dependencies

It says it's for Non-Debian system, so maybe it's only for me as an Arch Linux user?

I'll investigate!

Reply:
Looks like it's still required for me, I've added now a check which will set the `CPATH` when using a non-debian based Linux